src/modules/Data.js

```diff
@@ -1,88 +1,97 @@
-const backend_base = process.env.NEXT_PUBLIC_BACKEND_BASE_URL
-
+const backend_base = process.env.NEXT_PUBLIC_BACKEND_BASE_URL;
 
 export async function getSpecies(speciesId, token) {
     const result = await fetch(`${backend_base}/species/${speciesId}`, {
-        'method': 'GET',
-        'headers': {
-            Authorization: `Bearer: ${token}`
-        }
-    })
-    if (!result.ok) return -1
-    return await result.json()
+        method: 'GET',
+        headers: {
+            Authorization: `Bearer: ${token}`,
+        },
+    });
+    if (!result.ok) return -1;
+    return await result.json();
+}
+
+export async function getReqSpecies(species, token) {
```

Comment:
can u find a better name for this than ``getReqSpecies``? not very descriptive. maybe ``getSpeciesbyName`` or something

Reply:
getSpeciesByName done